bin/check-bundle-size.js

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
+
```

Comment:
We need some base license test here, or where it came from.

Reply:
I borrowed this file file from `debugger.html` so what should I write as license?

Reply:
you can just write where you borrowed the file, since we are not clear about its license.